src/services/avatarCache.ts

```
@@ -278,7 +373,354 @@ class AvatarCacheService {
+            for (let i = 0; i < localStorage.length; i++) {
+                const key = localStorage.key(i);
+                if (key && key.startsWith(CACHE_PREFIX)) {
+                    const value = localStorage.getItem(key);
+                    if (value) {
+                        try {
+                            const entry = JSON.parse(value) as CacheEntry;
+                            cacheEntries.push({
+                                key,
+                                timestamp: entry.timestamp
+                            });
+                        } catch (e) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)